src/test/java/org/jabref/logic/importer/fileformat/RISImporterTestFiles.java

```diff
@@ -1,58 +1,28 @@
 package org.jabref.logic.importer.fileformat;
 
 import java.io.IOException;
-import java.net.URISyntaxException;
-import java.nio.charset.StandardCharsets;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.List;
+import java.util.stream.Stream;
 
-import org.jabref.logic.bibtex.BibEntryAssert;
-import org.jabref.model.entry.BibEntry;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
 
-import org.junit.Assert;
-import org.junit.Before;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-import org.junit.runners.Parameterized.Parameter;
-import org.junit.runners.Parameterized.Parameters;
-
-@RunWith(Parameterized.class)
 public class RISImporterTestFiles {
 
-    private RisImporter risImporter;
-
-    @Parameter
-    public String fileName;
-
-    private Path risFile;
-
-
-    @Before
-    public void setUp() throws URISyntaxException {
-        risImporter = new RisImporter();
-        risFile = Paths.get(RISImporterTest.class.getResource(fileName + ".ris").toURI());
-    }
-
-    @Parameters(name = "{0}")
-    public static Collection<String> fileNames() {
-        return Arrays.asList("RisImporterTest1", "RisImporterTest3", "RisImporterTest4a", "RisImporterTest4b",
+    private static Stream<String> fileNames() {
+        return Stream.of("RisImporterTest1", "RisImporterTest3", "RisImporterTest4a", "RisImporterTest4b",
```

Comment:
It would be nice to have the RIS Importer collect the files dynamically like the other file tests do, e.g. starts with RIS and contains Test

Reply:
I will have a look at that and I will also refactor the remainder of the importer tests into this structure.

Reply:
I have extracted the relevant logic now for the RIS tests. However, it turns out that *the file reading logic in all importer tests is broken* (except for the RIS tests now). It just does not work, no files are found and in that case JUnit 4 didn't complain. JUnit 5 complains that there are no parameters. 

I am not sure if the importer tests never worked and people where just happy that they were green, or if they worked and got broken at some point in time. I'll gradually try to get the tests working now.